Looks like these are a header issue

```
Bridging.h:1:2: error: #import of type library is an unsupported Microsoft feature
# | #import "LocalCFunc.h"
```

I'll just add `UNSUPPORTED: OS=windows-msvc`

--- a/test/SourceKit/CodeComplete/complete_checkdeps_bridged.swift
+++ b/test/SourceKit/CodeComplete/complete_checkdeps_bridged.swift
@@ -1,10 +1,11 @@
-// UNSUPPORTED: OS=windows-msvc
 import ClangFW
 import SwiftFW
 
 func foo() {
   /*HERE*/
 }
+
+// UNSUPPORTED: OS=windows-msvc
 
 // RUN: %empty-directory(%t/Frameworks)
 // RUN: %empty-directory(%t/MyProject)

--- a/test/SourceKit/CodeComplete/complete_checkdeps_bridged.swift
+++ b/test/SourceKit/CodeComplete/complete_checkdeps_bridged.swift
@@ -1,3 +1,4 @@
+// UNSUPPORTED: OS=windows-msvc
 import ClangFW
 import SwiftFW
 

--- a/test/SourceKit/CodeComplete/complete_checkdeps_otherfile.swift
+++ b/test/SourceKit/CodeComplete/complete_checkdeps_otherfile.swift
@@ -1,10 +1,11 @@
-// UNSUPPORTED: OS=windows-msvc
 import ClangFW
 import SwiftFW
 
 func foo() {
   /*HERE*/
 }
+
+// UNSUPPORTED: OS=windows-msvc
 
 // RUN: %empty-directory(%t/Frameworks)
 // RUN: %empty-directory(%t/MyProject)

--- a/test/SourceKit/CodeComplete/complete_checkdeps_otherfile.swift
+++ b/test/SourceKit/CodeComplete/complete_checkdeps_otherfile.swift
@@ -1,3 +1,4 @@
+// UNSUPPORTED: OS=windows-msvc
 import ClangFW
 import SwiftFW
 

--- a/test/SourceKit/CodeComplete/complete_checkdeps_ownfile.swift
+++ b/test/SourceKit/CodeComplete/complete_checkdeps_ownfile.swift
@@ -1,3 +1,4 @@
+// UNSUPPORTED: OS=windows-msvc
 // BEGIN State1.swift
 import ClangFW
 import SwiftFW

--- a/test/SourceKit/CodeComplete/complete_checkdeps_ownfile.swift
+++ b/test/SourceKit/CodeComplete/complete_checkdeps_ownfile.swift
@@ -1,4 +1,3 @@
-// UNSUPPORTED: OS=windows-msvc
 // BEGIN State1.swift
 import ClangFW
 import SwiftFW

